.gitignore

```diff
@@ -27,3 +27,4 @@ ui/yarn.lock
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+config.json
```

Comment:
Is this for the UI or the backend?
I don't like the idea of a JSON config for the backend since there are blatantly other better tools for the job.

Reply:
backend, its just there temporarily.